### DISCUSSION TITLE
4090 only using half of the power when plotting Gigahorse?  90 sec plots?

### DISCUSSION THREAD

**Discussion by ChiaGigaFarmer** [madMAx43v3r/chia-gigahorse#137] [Q&A]
Hello.  I think im having a problem where my 4090 is only using half the power (per nvtop) and making 90 sec plots. I have a 4090 FE stripped with the EK waterblock on it.  Threadripper Pro 5955WX Processor,  EVGA Titanium 1600w power supply,  Asus Sage Pro WRX80 motherboard, 256GB 2Rx4 3200 ecc ram, Samsung enterprise nvme's for OS and Plot landing.  Plot landing is in mdadm raid 0, 4 drives. Linux OS. 22.04.  The only thing that ive done so far to try and fix it is enable gen4 on the pcie slot.

If anyone could help with this I would be much appreciative.

Thank you.

**Comment by hajes** [comment]
I do not understand your issue?

It looks like you have saturated your PC I/O, and there will be no improvements.

I have seen guys on EPYC servers with 3090, doing 80s.

**Comment by csuliuli** [comment]
Yes, due to the bandwidth limitation of  pcie, rtx4090 can not run with the max power. If you try to use two rtx4070 in the same syetem  with the -r 2 option , you will get the about 55 seconds for one plot file.
